---
title: Antimicrobial Resistance
layout: default
group: courses
---

Topics dealt with in the Advanced Genomics module held by 
**Prof. Matteo Brilli**
Twitter: @CompSysBiolUnit 
Twitter: @MatteoBip
Orcid: https://orcid.org/0000-0001-9891-4617
Website: https://comparativesystemsbiologygroup.github.io/CSB.github.io/


1. Advanced genomic techniques, bioinformatic processing and models (20 hours):
• ChIP-Seq and the like
– ChIP-Seq on epigenetic histone markers (in brief - made before);
– ChIP-Seq on bacterial transcription factors;
– Computational approaches for the reconstruction of gene regulatory networks with DNA pattern scanning (Information Theory, Position weight matrices) and gene expression com- pendia (CLR and the like) and the integration of ChIP-Seq data.
– Case studies (ChIP-Seq analysis with identification of transcription binding site within peaks, CLR on a gene expression compendium of E. coli).
• Tn-mutagenesis for the discovery of essential genes or genes involved in specific processes, with high fitness cost - experimental approaches (Tn-Seq, Tn-seq circle, TraDIS, Rb-Tn-seq...) and bioinformatic analysis
• Iso-seq for full transcriptome characterization;
• Introducing the sRNA world
– sRNA in Bacteria, bioinformatic prediction of gene coding sR- NAs, bioinformatic prediction of target transcripts. Experi- mental techniques (MAPS and RIL-Seq). Dynamical proper- ties of genetic circuits with sRNA-dependent regulation. Case studies: Quorum sensing regulation in Vibrio fischeri Fenley et al. [2011] and in Vibrio cholerae Teng et al. [2011].
– sRNAinPlantsandtheirpathogens,experimentalapproaches, sequencing and the quest for targets aided by PARE (de- gradome sequencing) in Plants. Immuno-precipitation of Ago1 protein. Bioinformatic approaches to target prediction. Phas- ing pattern identification. Mobile sRNAs in pathosystems. the Cuscuta case study.
• Metataxonomics, metagenomics,, metatranscriptomic for the char- acterization and the engineering of microbial communities e.g.
• Practical example: RNA-seq analysis to discover deaminase sig- natures (APOBEC, AIDAR) in viruses;
• Bonus advanced genomic techniques (to be defined). 2. Network Theory (8 hours)
• Definitions and concepts
• Distinctive characteristics of biological networks
• Topological analysis for the identification of essential/important genes, hubs and bottlenecks
• Network motifs and their dynamical properties
• The concept of community: characterization and resolution limit
• Epidemic spreading
• Robustness of biological networks;
3. Metabolic engineering (6 hours):
• Basics of metabolic networks and FBA modelling
• Optimizationofmetabolicprocessesforindustrialpurposes,Pareto front
• Practice on examples from publications;
4. Genome Wide Association Studies (8 hours)
• Quantitative Trait Loci in Plants and Animals e.g. 2
• Evolutionary genomics, the pangenome concept, orthologs, paralogs and orthogroups, phylogenetic profiles;
• Pathogens and the prediction of antibiotic resistance genes. Primer on machine learning techniques. e.g.
• Predictive models based on genomic properties, DNA compositional models.
5. Group Practical project on published data - a few proposals available to be discussed together!
